src/lib/components/workflow/workflows-summary-configurable-table/workflows-summary-configurable-table.svelte

```diff
@@ -444,7 +449,7 @@
   }
 
   .workflow-summary-table-wrapper.pinned.no-columns-pinned.batch-actions-visible {
```

Comment:
We might want to consider a solution for this overlap in the future

![Screenshot 2023-04-20 at 4 05 36 PM](https://user-images.githubusercontent.com/15069288/233498861-39fb3ff8-c61a-46fb-a42a-df2e5886c034.png)

Reply:
I think this is OK given the complexity of this table. Ideally we have less stuff in the table headers IMO 😂